Sources/Metric Logs/MetricViews.swift

```diff
@@ -0,0 +1,297 @@
+//
+//  MetricViews.swift
+//  DebugKit
+//  
+//  Copyright © 2022 Bottle Rocket Studios. All rights reserved.
+//
+
+import MetricKit
+import SwiftUI
+
+protocol MetricConfigurableView: View {
+    associatedtype Metric
+    init(payload: MetricPayload, metric: Metric)
+}
+
```

Comment:
All of these Metric views are really cool and useful. I could even see moving all of the MetricKit implementation to it's own repo.